### ISSUE TITLE
Theme is not correctly applied on specific parts of the UI

### ISSUE THREAD

**Issue by JMaravalhasSilva** [Birkbjo/OctoPrint-Themeify#144]
I'm running Themeify 1.2.2, Octoprint 1.8.7, and some specific parts of the UI are clearly not working as intended:

![image](https://user-images.githubusercontent.com/24859764/231843168-b58e56e0-b16d-4a68-b034-7612b62a4fe6.png)

![image](https://user-images.githubusercontent.com/24859764/231843365-5eb103fa-7b74-413d-b062-2d829fff1f99.png)

All settings at default values.

This issue appears to be somewhat similar to #118, but on different parts of the UI.

**Comment by LazeMSS** [comment]
I dont think this plugin is actively developed - tried contacting the developer but without any success. I have a similar plugin: https://plugins.octoprint.org/plugins/uicustomizer/ and was hoping to join forces with him.

**Comment by Birkbjo** [comment]
I actually spent some time last week on trying to migrate to using `css`-sheets for custom rules. So that should fix some of these issues. I can take a look at what you're doing and get some inspiration!

However this looks like just a missing/wrong css-rule in the theme, so can take a look at that.

**Comment by LazeMSS** [comment]
@Birkbjo  awesome ;) I was looking into the next branch also :) I was wondering if you had left the OP project. I don't have that much time and thought maybe we could join forces and help each other.

If I really had the time I would rewrite the UI but to be honest I have started using Klipper/Mainsail.

**Comment by Birkbjo** [comment]
> @Birkbjo awesome ;) I was looking into the next branch also :) I was wondering if you had left the OP project. I don't have that much time and thought maybe we could join forces and help each other.
> 
> If I really had the time I would rewrite the UI but to be honest I have started using Klipper/Mainsail.

I don't really have much time or motivation either, since the current version does what I need - getting a nice looking dark theme. 

I'm all in for that, looked briefly through the code. But it looks way more feature rich and complicated than Themeify.

Currently I just want to improve the customization, so instead of changing html styles, I just have a stylesheet object - eg https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleSheet. Ran into some issue that I couldn't have two, but I've a solution in mind.
